integration-tests/multinode/common.py

```diff
@@ -51,8 +51,6 @@ def wait_for_blocks(rpc, n):
 
 def stop_node(supervisor, name):
     supervisor.supervisor.stopProcess('%s:%s-%s' % (name, 'tendermint', name))
-    print('Wait 3 seconds before stop other processes[FIXME, remove after adr-001]')
```

Comment:
`integration-tests/multinode/byzantine_test.py:L27` and `integration-tests/multinode/jail_test.py:L47` also can removed now.

Reply:
ok